src/legacy/core_plugins/interpreter/common/types/kibana_datatable.ts

```diff
@@ -19,30 +19,49 @@
 
 import { map } from 'lodash';
 
+const name = 'kibana_datatable';
+
+interface Column {
+  id: string;
+  name: string;
+}
+
+interface Row {
+  [key: string]: object;
```

Comment:
I don't think `object` is the right type, here. A row is generally key: string, value: primitive, no? Maybe this should be: `[key: string]: unknown` or maybe any instead of unknown?

Reply:
oops, yes it is; thanks for catching that